SU-SWS/stanford_profile
# Several configuration tweaks.

# READY FOR REVIEW

# Summary
- Fixed an edge case where the gallery paragraph type would skip some images after you edit them
- Added a dedicated caption field instead of using the `title` attribute and displayed that caption in the display.
- Added fields to the gallery paragraph type to be similar to the teaser and lists
- Adjusted alot of help texts/labels
- Removed the sort by author name which was causing duplicates. We will have to come back to review that sorting criteria at a later date if so desired.

# Need Review By (Date)
- 2/4

# Urgency
- high

# Steps to Test
1. checkout this branch
2. `drush cim -y`
3. verify the changes have imported successfully.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
